.circleci/config.yml

```
@@ -410,7 +410,7 @@ workflows:
           requires: [build]
     triggers:
       - schedule:
-          cron: "10 20 * * 1"
+          cron: "45 10 * * 2"
```

```suggestion
          cron: "45 14 * * 2"
```
cron time is 4 hours ahead of EST